src/components/FAQSection/QuestionFAQ/QuestionFAQ.styles.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+import { Icon } from "@/components/Icon/Icon";
+import styled from "styled-components";
+
+export const SubSectionFAQ = styled.p`
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  transition: 100ms;
+  padding: 0 1rem;
+  margin: 0;
+`;
+
+export const IconWrapper = styled(Icon)`
+  color: ${({ theme }) => theme.colors.black};
+  transition: 100ms linear;
+  flex-shrink: 0;
+`;
+
+export const Summary = styled.summary`
+  display: flex;
+  justify-content: space-between;
+  font-weight: bold;
+  cursor: pointer;
+  border-radius: 4px;
+  padding: 1rem;
+  align-items: center;
+`;
+
+export const QuestionContainer = styled.details`
+  display: flex;
+  width: 100%;
```

Comment:
```suggestion
  width: 100%;
  max-width: 60rem;
```

I saw that you added a width: 50% to make it more responsive, I think that it's more fluid to add just this max-width here and remove the width: 50%;

Reply:
really nice, i'll do that